### PR TITLE
COMMERCE-5045 reorganize dispatch file repository

### PR DESCRIPTION
**Pull request by igorbeslic**

### CODE REVIEW

**Comment by igorbeslic** [comment]
ci:forward

File: modules/apps/dispatch/dispatch-service/src/main/java/com/liferay/dispatch/internal/repository/DispatchFileValidatorImpl.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+@Component(
+	configurationPid = "com.liferay.dispatch.configuration.DispatchConfiguration",
+	immediate = true,
+	property = "dispatch.file.validator.type=" + DispatchConstants.REPOSITORY_DEFAULT_FILE_VALIDATOR_TYPE,
+	service = DispatchFileValidator.class
+)
+public class DispatchFileValidatorImpl extends BaseDispatchFileValidator {
+
+	@Override
+	public void checkExtension(String fileName)
```
Comment (dsitu) [inline modules/apps/dispatch/dispatch-service/src/main/java/com/liferay/dispatch/internal/repository/DispatchFileValidatorImpl.java:43]: @igorbeslic wondering if `validateFileExtension` would be a better name.

File: modules/apps/dispatch/dispatch-service/src/main/java/com/liferay/dispatch/internal/repository/DispatchFileValidatorImpl.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+		if (contains(_dispatchConfiguration.fileExtensions(), extension)) {
+			return;
+		}
+
+		throw new DispatchRepositoryException(
+			"Invalid file extension for " + fileName);
+	}
+
+	@Override
+	public void checkSize(long size) throws DispatchRepositoryException {
```
Comment (dsitu) [inline modules/apps/dispatch/dispatch-service/src/main/java/com/liferay/dispatch/internal/repository/DispatchFileValidatorImpl.java:57]: Maybe `validateFileSize`?

**Comment by igorbeslic** [comment]
ci:test:sf

**Comment by igorbeslic** [comment]
ci:forward

**Comment by igorbeslic** [comment]
ci:stop

**Comment by igorbeslic** [comment]
ci:forward

**Comment by igorbeslic** [comment]
ci:forward

**Comment by igorbeslic** [comment]
ci:forward

**Pull request closed without merging**